vortex-duckdb/src/duckdb/value.rs

```diff
@@ -71,8 +71,14 @@ impl Value {
         unsafe { Self::own(cpp::duckdb_create_date(cpp::duckdb_date { days })) }
     }
 
-    pub fn as_string(&self) -> &CStr {
-        unsafe { CStr::from_ptr(cpp::duckdb_get_varchar(self.as_ptr())) }
+    pub fn as_string(&self) -> String {
```

Comment:
IMO you should return a CString and allow the caller to decide how to handle utf-8 validation errors.

Reply:
Eh, nah, this is more practical!